README.md

```diff
@@ -95,7 +95,8 @@ Please refer [vDVS configuration page](http://vmware.github.io/docker-volume-vsp
    - Needs [open vm tools or VMware Tools installed](https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=340) ```sudo apt-get install open-vm-tools```
 - RedHat and CentOS
 - Windows Server 2016 (64 bit)
-- [Photon 1.0, Revision 2](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-10-revision-2-binaries) (v4.4.51 or later) [Photon 2.0] (https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-20-ga-binaries)
+- [Photon 1.0, Revision 2](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-10-revision-2-binaries) (v4.4.51 or later)
+- [Photon 2.0](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-20-ga-binaries)
```

Comment:
it should be in the same line ... either you may add [Photon 1.0, Revision 2](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-10-revision-2-binaries) (v4.4.51 or later) and [above](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-20-ga-binaries) or in the same line [Photon 1.0, Revision 2](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-10-revision-2-binaries) (v4.4.51 or later), [Photon 2.0](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-20-ga-binaries)